### DISCUSSION TITLE
New version - Test

### DISCUSSION THREAD

**Discussion by laredson** [kldzj/pzmod#12] [General]
Hello!

I've noticed you're upgraded to a new version, if you want to test something... not sure this is an actual release or just a minor update

**Comment by kldzj** [comment]
Hey,

yes, I released the current state as final `v2.0.0`. So no more pre-releases for `v2.0.0`. This does not mean development has stopped, missing features or fixes will get released as `v2.0.1`, `v2.1.0` or maybe even pre-releases again. :)

The only major change for you with this update would be that it's asking you per map entry where to put it.

**Comment by laredson** [comment]
I'm trying to build a new server from a collection. It still asks me once per mod even if the mod has only 1 entry. 
I thought you solved this, didn't you?

**Reply by kldzj** [reply]
No, didn't do that yet, issue #6 is still open. I'll get to that in the next days. :)

**Reply by laredson** [reply]
Ahh sorry I misunderstood.
I hope it comes soon!
